plugins/modules/zos_stat.py

```diff
@@ -0,0 +1,1685 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) IBM Corporation 2025
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+---
+module: zos_stat
+version_added: '1.14.0'
+author:
+  - "Ivan Moreno (@rexemin)"
+short_description: Retrieve facts from z/OS
+description:
+  - The L(zos_stat,./zos_stat.html) module retrieves facts from resources
+    stored in a z/OS system.
+  - Resources that can be queried are files, data sets and aggregates.
+options:
+  name:
+    description:
+        - Name of a data set or aggregate, or a file path, to query.
```

Comment:
This sounds a bit weird to me, maybe we can change it to something like
```suggestion
        - Name of a data set, aggregate, or a file path to query.
```